### ISSUE TITLE
Add testing via jest

### ISSUE THREAD

**Issue by dkundel** [twilio-labs/actions-sms#5]
We should add some basic testing using [jest](https://jestjs.io). This might require some refactoring to make the code a bit more testable.

**Comment by dangilkerson** [comment]
I'd like to work on this

**Comment by dkundel** [comment]
That would be great :) let me know if you need anything from me!

**Comment by dangilkerson** [comment]
Is there a reason node_modules are checked in?

**Comment by dkundel** [comment]
Yes those are the production dependencies only. It's a GitHub actions thing. i described it in #6

**Comment by dangilkerson** [comment]
I exported a function in main to test since I wanted to minimize the changes in case there were other changes on going. Probably should pull the code out of main.js when things are stable.
